Cargo.toml

```diff
@@ -1,36 +1,30 @@
 [package]
 name = "forc-wallet"
-version = "0.12.0"
+version = "0.13.0"
```

Comment:
Should we bump in this one, or cut the release separately?

Reply:
Yes, this way there's one less PR.